### PR TITLE
Lrouhier/update readme

### PR DESCRIPTION
**Pull request by lrouhier**
New changes after reviews

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,10 +57,12 @@ Installation
 #. Install Docker:
 
    - If you are on Windows 10 Pro/Enterprise: Install `Docker <https://store.docker.com/editions/community/docker-ce-desktop-windows/>`_ 
-   
+
+   See this `tutorial  <https://docs.docker.com/docker-for-windows/install/>`_ to install Docker.
```
Comment (jcohenadad) [inline README.rst]: i would indent this with the same indentation as the previous bullet, so it is more intuitive that this tutorial refers to the previous line

File: README.rst [hunk 2]
```diff
@@ -57,10 +57,12 @@ Installation
...
+
    - If you are on Windows XP/VISTA/7/8/8.1/10 others than Pro/Enterprise: Install `Docker Toolbox <https://docs.docker.com/toolbox/overview/>`_.
 
-   See this `tutorial <https://docs.docker.com/toolbox/toolbox_install_windows/>`_ to install Docker.
+   See this `tutorial <https://docs.docker.com/toolbox/toolbox_install_windows/>`_ to install Docker Toolbox.
```
Comment (jcohenadad) [inline README.rst]: same comment about the identation

File: README.rst [hunk 3]
```diff
@@ -149,42 +151,46 @@ Usage
...
-    - the docker_shared_folder should be highlighted in green : 
-	.. image:: /picture/screenshot_green.PNG
+    - the docker_shared_folder should be highlighted in green :
+ 
+	.. image:: picture/screenshot_green.PNG
 	
       if not : 
-	- check the permission on your local host file :
-	    - go to C:/Users/
+      - check the permission on your local host file :
```
Comment (jcohenadad) [inline README.rst]: wrong formatting of your bullet point

File: README.rst [hunk 4]
```diff
@@ -194,11 +200,11 @@ Usage
```
Comment (jcohenadad) [inline README.rst:201]: shouldn't the "- Open VirtualBox GUI " and the next bullet point be indented at the same level as "- Stop Docker Machine if..."?

Comment (jcohenadad) [reply README.rst:201]: the "Restart Docker Machine, by running command :" also seems wrongly indented

**Review by jcohenadad — CHANGES_REQUESTED** [review]
a few issues (see my single comments). I suggest you use the feature "preview" which gives you a preview of the formatted rst

**Comment by lrouhier** [comment]
I hope it's better. Sorry about the number of commit, it's easier to see the display on github.com

**Review by jcohenadad — APPROVED** [review]

**Pull request merged**